sys/log/common/include/log_common/log_common.h

```diff
@@ -96,6 +96,7 @@ struct log;
 #define LOGS_NMGR_OP_LEVEL_LIST   	(4)
 #define LOGS_NMGR_OP_LOGS_LIST    	(5)
 #define LOGS_NMGR_OP_SET_WATERMARK	(6)
+#define LOGS_NMGR_OP_MODLEVEL	        (8)
```

Comment:
What happened to 7? :)

Reply:
oops, sorry about that. I had two separate IDs earlier and I deleted the 7 instead of the 8.